worldwind/src/commonMain/kotlin/earth/worldwind/shape/Path.kt

```diff
@@ -52,6 +54,20 @@ open class Path @JvmOverloads constructor(
         protected fun nextCacheKey() = Any()
     }
 
+    override fun addToBatch(rc : RenderContext) : Boolean {
+        val canBeBatched = super.addToBatch(rc) && (!isExtrude || isSurfaceShape) && activeAttributes.interiorImageSource == null
```

Comment:
Could you comment in the code why we exclude extrude and interior images, please

Reply:
done

Reply:
done